docs/guide/getting_started.md

```diff
@@ -1,6 +1,94 @@
 ---
-title: Getting Started 
+title: Getting Started
 weight: -10
 ---
 
 ## Getting Started
+
+[Kubernetes](https://kubernetes.io/) is an open-source system for automating
+deployment, scaling, and management of containerized applications.
+
+The "Kubernetes as a Service" offering on Converged Cloud (Codename: Kubernikus)
```

Comment:
CCloud makes no sense for outside readers. I think, we need to generalise or remove those references.